Automattic/sensei
# Fix site logo getting distorted

Resolves https://github.com/Automattic/sensei/issues/7271

## Proposed Changes
* Recently we increased the priority (Set the priority of the base learning mode styles to 9 instead of default 10) of the learning mode CSS file so that the theme specific learning mode CSS files can override the base CSS styles without the need to make the styles specify more basic learning mode styles.

The problem was the styles that come from wp core, or Gutenberg or the themes (not referring to the theme specific CSS we have) were still getting loaded with priority 10. So these core/gb/other styles were also overriding our LM styles if the specificity was same.

For example, in this particular instance with site icon, we have a style to set the width of site logo to auto, in core the width for that is 120px. So core style was overriding ours.

We could just remove the priority 9, but that'd have removed the added facility we got, which is overriding base lm styles with theme specific LM styles.

To still have that facility, we removed the 9, but added 11 to the theme specific styles. Now our base LM styles overrides the core styles, and our theme specific styles override the base LM styles.

Also, there was additional call to Course theme's enqueue function from divi's PHP file. We've removed that.

Before increasing priority:
![Screenshot 2023-11-07 at 7 50 39 PM](https://github.com/Automattic/sensei/assets/6820724/6c8a9901-9aa1-431c-bb5b-fcd1f19737f3)

After increasing priority:
![Screenshot 2023-11-07 at 7 51 38 PM](https://github.com/Automattic/sensei/assets/6820724/082ea18b-0288-4e04-bc84-95ece3e3c206)

## Testing Instructions


1. Install one of those themes Astra, GeneratePress, Storefront
2. Add site logo in Appearance > Customize > Site Identitiy (add a square logo and skip cropping)
3. Visit a lesson in the Learning Mode
4. Make sure the site logo is not distorted.
5. Now add the following style in main `learning-mode.css` file
```
.sensei-course-theme .wp-block-site-logo img {
	max-height: 124px;
	width: 150px;
}
```
6. Check in the frontend, make sure the icon has grown to 150px
7. Now without removing the above style, add the following style in Astra's (assuming you are using Astra) learning-mode.css (assets/css/3rd-party/themes/astra/learning-mode.css)
```
.sensei-course-theme .wp-block-site-logo img {
	max-height: 124px;
	width: 10px;
}
```
8. Make sure the theme specific style is overriding the base LM style
9. Do not push the CSS changes :p

## Pre-Merge Checklist
